Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -427,7 +427,7 @@ public void EnsureValid(string[] sponsorMarkings)
             var configManager = IoCManager.Resolve<IConfigurationManager>();
             if (configManager.GetCVar(CCVars.RestrictedNames))
             {
-                name = Regex.Replace(name, @"[^А-Я,а-я,0-9, -]", string.Empty); // Corvax: Only cyrillic names
+                name = Regex.Replace(name, @"[^А-Я,а-я,ё,Ё,0-9,', -]", string.Empty); // Corvax: Only cyrillic names
```

Comment:
```suggestion
                name = Regex.Replace(name, @"[^\u0430-\u04510-9'-]", string.Empty); // Corvax: Only cyrillic names
```
Может тогда просто использовать юнидод рендж? еще и запятые какие-то непонятные были

Reply:
Если меня закоммитишь, то в ченджлоге напиши, что ТЕПЕРЬ ЗАПЯТУЮ ИСПОЛЬЗОВАТЬ ЗАПРЕЩЕНО